src/vkd3d-proton/vkd3d-proton_interfaces.h

```diff
@@ -29,7 +29,13 @@
 
 enum D3D12_VK_EXTENSION : uint32_t {
     D3D12_VK_NVX_BINARY_IMPORT = 0x1,
-    D3D12_VK_NVX_IMAGE_VIEW_HANDLE = 0x2
+    D3D12_VK_NVX_IMAGE_VIEW_HANDLE = 0x2,
+    D3D12_VK_NV_LOW_LATENCY_2 = 0x3
```

Comment:
This seems to be both unused and unnecessary, seeing as we can already query extension support via `ID3DLowLatencyDevice::SupportsLowLatency`. But I guess the validity of this comment depends on whether maintainers of DXVK and vkd3d-proton agree that `ID3DLowLatencyDevice::SupportsLowLatency` is the way to go.

Reply:
I only added this to have a complete list of supported NV extensions. Agreed that it isn't necessary though. I will wait for the vkd3d-proton feedback before removing it though.